lib/lambda/sinkMainProcessors.ts

```diff
@@ -146,7 +146,7 @@ const getMakoDocTimestamps = async (kafkaRecords: KafkaRecord[]) => {
   const openSearchRecords = await getItems(kafkaIds);
 
   return openSearchRecords.reduce<Map<string, number>>((map, item) => {
-    if (item.changedDate !== null) {
+    if (item.changedDate !== undefined) {
```

Comment:
maybe do it as  if(item?.changedDate){}.  Woulc catch null and undefined unless we want to do null differently